app/routes/articles.js

```diff
@@ -193,7 +194,7 @@ function articleListingPromise(key, identifier, isReviewing = false) {
     const category = __validCategory(key)
 
     if(!category) throw new Error(`Invalid Category: ${category}`)
-    if(category !== ALL && !identifier) throw 'Invalid Query'
+    if(category !== ALL && !identifier) throw new Error('Invalid Query')
```

Comment:
Could u put here the same thing as in `if (!category)`, to make error tell u what value is category's one. I actually wanted you to do it for both conditions the last time, but maybe I didn't convey it well enough

Reply:
Sure